resources/src/app.css

```diff
@@ -95,3 +95,8 @@ textarea {
   padding: 10px 14px;
   resize: vertical;
 }
+
+.row {
```

Comment:
Is it possible to add the `display: flex` rule to some other selector, instead of creating a whole new utility class for this?